### PR TITLE
tests: cancel payment request, submit button disabled

### LINKED ISSUE #94: Test cancel payment request for a linked proposal
Add playwright test case for canceling a payment request when there is a linked proposal

### LINKED ISSUE #95: Test that submit should be disabled when incorrect receiver id is mentioned or empty amount or empty proposal name or empty token
Create a Playwright test for payment requests that submit should be disabled when incorrect receiver id is mentioned or empty amount or empty proposal name or empty token

### PR DESCRIPTION
**Pull request by petersalomonsen**
add two of the remaining test cases after the QA effort in #63 

- cancel payment request for a linked proposal 

https://github.com/user-attachments/assets/17100b01-06c4-4d09-8edb-68b2ac3229ec

- submit should be disabled when incorrect receiver id is mentioned or empty amount or empty proposal name or empty token

https://github.com/user-attachments/assets/ec704010-0697-4b5b-9770-af294cf030ba

- also added util methods `focusInputClearAndBlur` and `focusInputReplaceAndBlur` for simulation of clearing and input and filling in a new value
- added util method for mocking the NEAR price

fixes #94 
fixes #95

### CODE REVIEW

**Comment by petersalomonsen** [comment]
@race-of-sloths include

**Comment by race-of-sloths** [comment]
@petersalomonsen Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!


[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://race-of-sloths.com/images/bot-banner-dark.svg">
    <source media="(prefers-color-scheme: light)" srcset="https://race-of-sloths.com/images/bot-banner-light.svg">
    Shows inviting banner with latest news.
</picture>](https://race-of-sloths.com/latest-news)

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/petersalomonsen?type=bot&pr=NEAR-DevHub/neardevhub-treasury-dashboard/93&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/petersalomonsen?type=bot&pr=NEAR-DevHub/neardevhub-treasury-dashboard/93&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/petersalomonsen)


<details><summary>Current status: <i>waiting for scoring</i></summary>

We're waiting for maintainer to score this pull request with `@race-of-sloths score [0,1,2,3,5,8,13]` command. Alternatively, autoscoring [1,2] will be applied for this pull request

</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/petersalomonsen)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Invite contributor** `@race-of-sloths invite` to invite the contributor to participate in a race or include it, if it's already a runner.
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

**Review by Megha-Dev-19 — APPROVED** [review]
LGTM

**Comment by Megha-Dev-19** [comment]
@race-of-sloths score 5

**Comment by race-of-sloths** [comment]
🥁  __Score it!__

@Megha-Dev-19, please score the PR with `@race-of-sloths score [1/2/3/5/8/13]`. The contributor deserves it.
If no scoring is provided within 24 hours, this PR will be scored as 2 🦥

**Comment by Megha-Dev-19** [comment]
@race-of-sloths score 5

**Pull request merged**
